WalletWasabi.Documentation/HowToDeploy.md

```diff
@@ -23,15 +23,24 @@
 
    Note: As you can see, the deployer can choose the `codename` freely.
 
-2. Deploy using the `deploy.sh` script.
+   - Creating a tag: `git tag v2.0.3-myexampletag`
+   - Push the tag to upstream: `git push upstream v2.0.3-myexampletag`
```

Comment:
I never pushed the tag to the repo before and Lucas adviced me not to do that, as we only need it locally.
Is this still true @lontivero?

Reply:
Hmm if I am not pushing the tag how the backend will get it? Do you execute the git tag command on the backend itself?

Reply:
I think the tag is unnecessary. Right now we need them because the `deploy.sh` requires them but that will be changed. see: https://github.com/zkSNACKs/BackendScripts/pull/18